config/environments/development.rb

```diff
@@ -75,7 +75,7 @@
   config.assets.raise_runtime_errors = true
 
   # Only log info and higher on development
-  config.log_level = :info
+  config.log_level = :debug
```

Comment:
I also use this locally a lot, but I'm not sure if this is needed to be changed?

Reply:
I don't know, it's default value in Rails 5